ros2topic/ros2topic/api/__init__.py

```diff
@@ -259,3 +262,64 @@ def add_qos_arguments(
             f'Quality of service liveliness lease duration setting to {entity_type} '
             'with (overrides liveliness lease duration value of --qos-profile option, default: '
             f'{default_profile.liveliness_lease_duration})'))
+
+
+def choose_qos(node, topic_name: str, qos_args):
```

Comment:
> [!NOTE]  
> Although it is okay to move this from `echo.py`, downstream packages are affected. since this change is only for rolling that is also fine too.